Drop-In/src/main/java/com/braintreepayments/api/dropin/adapters/SupportedPaymentMethodsAdapter.java

```diff
@@ -23,19 +24,20 @@
     private ArrayList<PaymentMethodType> mAvailablePaymentMethods;
     private PaymentMethodSelectedListener mPaymentMethodSelectedListener;
 
-    public SupportedPaymentMethodsAdapter(Context context, Configuration configuration,
-                                          boolean androidPayEnabled, boolean unionpaySupported,
+    public SupportedPaymentMethodsAdapter(Context context,
```

Comment:
This is technically a breaking change, even though I don't think a of folks would be using this adapter.